static/js/platformDropdowns.js

```diff
@@ -39,19 +42,14 @@ function setHeaderDropdowns($uniqueness) {
   }
 }
 
-/*function setPlatformCookie($platform) {
-  Cookies.set('platform', $platform, { expires: $cookieLife });
-}*/
-
-function setPlatformCookie($platform, $product, $title) {
+function setPlatformCookie($platform) {
   Cookies.set('platform', $platform, { expires: $cookieLife });
   togglePlatformDivs();
   setTopDropdown();
-  $platform = $platform.replace("/", "_");
-  $platform = $platform.replace(" ", "-");
-  console.log($platform)
-  var dropdown = document.getElementById("platformButtonTitleText-" + $platform);
-  dropdown.innerHTML = Cookies.get("platform") + '<i class="fa fa-chevron-down dropdownArrow-' + $platform + '" aria-hidden="true"></i>';
-  dropdown.scrollIntoView();
```

Comment:
This 6 lines were causing an error in the browser console because `dropdown` was `null`, so I simply removed it.